Kitodo/src/test/java/org/kitodo/production/services/data/TaskServiceIT.java

```diff
@@ -216,8 +216,7 @@ public void shouldGetTitleWithUserName() throws Exception {
 
     @Test
     public void shouldGetCurrentTasksOfBatch() {
-        List<Task> tasks = taskService.getCurrentTasksOfBatch("Task", 1);
-        System.out.println("shouldGetByTitleAndBatches: " + tasks.size());
+        taskService.getCurrentTasksOfBatch("Task", 1);
     }
```

Comment:
There is no test at all or what is this test is checking for? Even old code did not test anything :-(

Reply:
I was just removing the system out. But you are right, i'm adding a check

Reply:
Ok, there is something wrong with this method. I'm going to fix it in a separat Pull Request